DNN Platform/Library/Services/Zip/SharpZipLibRedirect.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+
+namespace DotNetNuke.Services.Zip
+{
+    /// <summary>
+    /// This class is in charge of fixing an assembly rename which was necessary when DNN
+    /// upgraded from an older SharpZipLib. 
+    /// </summary>
+    /// <remarks>
+    /// final solution taken from https://raw.githubusercontent.com/2sic/2sxc/master/2sxc%20Dnn/Dnn920/SharpZipLibRedirect.cs
+    /// read the readme.md in this folder for more info
+    /// </remarks>
+    public class SharpZipLibRedirect
```

Comment:
I'm not real excited about this being `public`, seems like it will confuse folks that there's a `DotNetNuke.Services.Zip` namespace that doesn't have anything useful for them inside of it.  Should this just live next to (or inside of) `DotNetNukeHttpApplication`?

Reply:
I agree, internal would be better if we can

Reply:
ok great suggestion!